lightsaber.ino

```diff
@@ -540,6 +540,10 @@ public:
     CONFIG_VARIABLE(Transition1Degrees, 45.0f);
     CONFIG_VARIABLE(Transition2Degrees, 160.0f);
     CONFIG_VARIABLE(MaxSwingVolume, 3.0f);
+    CONFIG_VARIABLE(AccentSwingSpeedThreshold, 0.0f);
+    CONFIG_VARIABLE(AccentSwingVolumeSharpness, 1.0f);
```

Comment:
I think we should move these variables (except for AccentSwingSpeedThreshold) into config.ini, which can be used directly from hybrid font. That way they can be used to adjust swings for non-smoothswing fonts as well.

Reply:
ok, i'll drop 'accent' from the parameter names and move them to config.ini  That way we're adjusting 'swing' parameters in config.ini and only calling 'accent' swings from smoothsw.ini

Reply:
config.ini is the same file that Igniter uses, so we need to use some sort of naming that makes it clear that these variables are intended for ProffieOS. Simplest way is probably to just use replace "Accent" with "ProffieOS". Hopefully Igniter ignores variables it doesn't recognize.

Reply:
done, moved variables to config.ini